addon/services/building.js

```diff
@@ -72,7 +72,10 @@ export default class BuildingService extends GwrService {
 
     if (!response.ok) {
       const xmlErrors = await response.text();
-      const errors = this.extractErrorsFromXML(xmlErrors);
+      const errors = this.extractErrorsFromXML(xmlErrors, [
+        Building.FORBIDDEN_BUILDING_CLASS_ERROR,
+        this.intl.t("ember-gwr.building.forbiddenBuildingClassError"),
+      ]);
```

Comment:
I think in theory it is allowed in updates, but let's leave it. It doesn't show if there is no error.